apps/remote/src/Application.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react';
+import { RemoteClient } from './RemoteClient';
+import MainView from './MainView';
+import { ModalWrapper } from './modals/ModalWrapper';
+
+let remoteClient: RemoteClient | undefined;
+function initializeClient() {
+  if (!remoteClient) {
+    remoteClient = new RemoteClient(`ws://${window.location.hostname}:8380/interfaces/v1/remote-app`);
```

Comment:
> remoteClient = new RemoteClient(`ws://${window.location.hostname}:8380/interfaces/v1/remote-app`);

Wouldn't it better to use `${this.serverConf.port}` for the port incase user has a different port set.